packages/atomic/cypress/e2e/facets/facet-depends-on/facet-depends-on.cypress.ts

```diff
@@ -134,7 +133,7 @@ describe('Facet DependsOn Test Suites', () => {
 
         it('should not have a duplicate history state', () => {
           cy.go('back');
-          cy.location('href').should('eq', 'about:blank');
+          cy.location('pathname').should('eq', '/');
```

Comment:
https://github.com/coveo/ui-kit/pull/3846/files/890528bf25d988e017edac2561cd5c29ed98448f#r1577259872 ... and so blank is now pathname=/, it does the same